app/views/modules/_header.html.erb

```diff
@@ -10,31 +10,57 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<div id="masthead" class="container">
-  <div class="row visible-xs">
-    <div class="lib-logo col-xs-8">
-      <%= ualib_logo_link %>
-    </div>
-  </div>
-  <div class="row">
-    <div class="lib-logo col-sm-3 hidden-xs">
-      <%= ualib_logo_link %>
-    </div>
-    <div class="mast-branding col-sm-5">
-      <%= era_av_logo_link %>
+<div class="container-fluid grey">
+
+  <div id="masthead" class="container">
+    <div class="row visible-xs">
+      <div class="lib-logo col-xs-8">
+        <%= ualib_logo_link %>
+      </div>
+      <div class="col-xs-4 user-util">
+          <%= render partial: "modules/user_util_dropdowns"%>
+      </div>
     </div>
-    <div class="col-sm-4">
-      <form accept-charset="UTF-8" action="/catalog" id="search-form-header" method="get" role="search" class="form-inline hidden-xs mast-sm-search">
-        <div style="display:none"><input name="utf8" type="hidden" value="✓"></div>
-        <input id="search_field" name="search_field" type="hidden" value="all_fields">
-        <div class="form-group">
-          <input type="text" class="form-control q" id="search-field-header" name="q" placeholder="enter search terms">
-        </div>
-        <button type="submit" class="btn btn-default sm-search"><span class="glyphicon glyphicon-search" aria-hidden="true"></span>Search ERA Audio+Video</button>
-      </form>
-      <div class="col-md-12 user-util hidden-xs">
-        <%= render partial: "modules/user_util_dropdowns"%>
+    <div class="row">
+      <div class="lib-logo col-sm-4 hidden-xs">
+        <%= ualib_logo_link %>
+        <%= link_to "ERA Home", "https://era.library.ualberta.ca", :class => "era-link" %>
       </div>
+        <div class="mast-branding col-sm-4">
+          <h2 class="hidden-xs">education &amp; research archive</h2>
```

Comment:
wait sorry - I'm wrong. It should not be hidden. I've fixed this in the favicon PR.